src/main/scala/com/fulcrumgenomics/umi/CollectDuplexSeqMetrics.scala

```diff
@@ -58,63 +59,64 @@ object CollectDuplexSeqMetrics {
-    *   2. SS or "Single Strand": single-strand families that are each subsets of a CS family create by
-    *      also using the UMIs to partition the larger family, but not linking up families that are
-    *      created from opposing strands of the same source molecule.
-    *   3. DS or "Double Strand": families that are created by combining single-strand families that are from
-    *      opposite strands of the same source molecule. This does NOT imply that all DS families are composed
-    *      of reads from both strands; where only one strand of a source molecule is observed a DS family is
-    *      still created.
+    * Metrics produced by `CollectDuplexSeqMetrics` to quantify the distribution of different kinds of read family
+    * sizes.  Three kinds of families are described:
     *
-    * @param family_size the family size, i.e. the number of read pairs grouped together into a family
-    * @param cs_count the count of families, of family size, when grouping just by coordinates and strand information
-    * @param cs_fraction the fraction of all CS families where size == family_size
-    * @param cs_fraction_gt_or_eq_size the fraction of all CS families where size >= family_size
-    * @param ss_count the count of families, of family size, when also grouping by UMI to create single-strand families
-    * @param ss_fraction the fraction of all SS families where size == family_size
-    * @param ss_fraction_gt_or_eq_size the fraction of all SS families where size >= family_size
-    * @param ds_count the count of families, of family size, when also grouping by UMI and merging single-strand
-    *                 families from opposite strands of the same source molecule
-    * @param ds_fraction the fraction of all DS families where size == family_size
-    * @param ds_fraction_gt_or_eq_size the fraction of all DS families where size >= family_size
+    * 1. _CS_ or _Coordinate & Strand_: families of reads that are grouped together by their 5' genomic
+    *    positions and strands just as they are in traditional PCR duplicate marking
+    * 2. _SS_ or _Single Strand_: single-strand families that are each subsets of a CS family create by
+    *    also using the UMIs to partition the larger family, but not linking up families that are
+    *    created from opposing strands of the same source molecule.
+    * 3. _DS_ or _Double Strand_: families that are created by combining single-strand families that are from
+    *    opposite strands of the same source molecule. This does **not** imply that all DS families are composed
+    *    of reads from both strands; where only one strand of a source molecule is observed a DS family is
+    *    still created.
+    *
+    * @param family_size The family size, i.e. the number of read pairs grouped together into a family.
+    * @param cs_count The count of families, of family size, when grouping just by coordinates and strand information.
```

Comment:
This was a bit confusing.
"The count of families, of family size," -> "The count of families of a given family size"